docs/security.md

```diff
@@ -37,26 +40,35 @@ Both stores must be base64 encoded, for example:
 
 **Note:** The base64 string of the keystore will probably be much longer than the snippet above, spanning 50 lines or so.
 
-With this and the password `secret` for the keystore and the private key, your JSON options file will look like this:
+Add the stores to your secrets in the DC/OS Secret store, for example if your base64 encoded keystores and truststores are server.jks.base64 and trust.jks.base64, respectively then do the following: 
+
+```bash
+dcos security secrets create /truststore --value-file trust.jks.base64
+dcos security secrets create /keystore --value-file server.jks.base64
+```
 
-    {
-      "security": {
-        "ssl": {
-          "enabled": true,
-          "keyStoreBase64": "/u3+7QAAAAIAAAACAAAAAgA...”,
-          "keyStorePassword": "secret",
-          "keyPassword": "secret"
-        }
-      }
-    }
+In this case you're adding two secrets `/truststore` and `/keystore` that you will need to pass to the Spark Driver and Executors. You will need to add the following configurations to your `dcos spark run ` command:
```

Comment:
In this case, you are adding....

You must add the following configurations...